bom/pom.xml

```diff
@@ -384,6 +429,11 @@
                 <artifactId>cxf-rt-security</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
```

Comment:
This is one is certainly should not be there, this is internal OSGi bundle and should not be consumed by applications or services directly.

Reply:
Removed cxf-servlet-compatible